Source/WebCore/bindings/js/SerializedScriptValue.cpp

```diff
@@ -1116,7 +1117,7 @@ class CloneSerializer : CloneBase {
             return;
         }
 
-        auto arrayBuffer = pixelBuffer->data().possiblySharedBuffer();
+        auto arrayBuffer = downcast<Uint8ClampedPixelBuffer>(*pixelBuffer).data().possiblySharedBuffer();
```

Comment:
Yes it should be of type ByteArrayPixelBuffer since we are passing the default ImageBufferAllocator to getPixelBuffer(). But I changed the if-statement which comes before this code from` if (!pixelBuffer) `to `if (!is<ByteArrayPixelBuffer>(pixelBuffer))` to make it clear that this is what we expect.